cmd/noms/splore/js2go.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+import sys
+
+def main():
+    if len(sys.argv) != 4:
+        print('usage: %s <source> <package> <varname>' % (sys.argv[0]))
+        return
+
+    jsf, pkg, varname = sys.argv[1:]
+
+    sys.stdout.write('package %s\n\n' % (pkg,))
+    sys.stdout.write('const %s = "' % (varname,))
```

Comment:
Because there are backticks in the JS source code, and in Go you can't have a backtick inside a backtick.